exercises/concept/annalyns-infiltration/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+pub fn can_fast_attack(_knight_is_awake: bool) -> bool {
+    unimplemented!("Implement can_fast_attack");
+}
+
+pub fn can_spy(_knight_is_awake: bool, _archer_is_awake: bool, _prisoner_is_awake: bool) -> bool {
```

Comment:
```suggestion
pub fn can_spy(knight_is_awake: bool, archer_is_awake: bool, prisoner_is_awake: bool) -> bool {
```

Reply:
I think this is off-topic for this PR. Both approaches are currently being used on the track. I agree we should make this consistent and introduce some CI test to make sure it *stays* consistent, but this PR is not the place to do it. Let's keep the underscores for now.